### PR TITLE
chore: Remove 'About' section of event

### LINKED ISSUE #1880: Remove 'About' section of event
**Describe the chore**
Remove 'About' section of event.

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1880 

Changes:

Removed `About` section of event.

### CODE REVIEW

File: app/src/main/java/com/eventyay/organizer/core/main/DrawerNavigator.java [hunk 1]
```diff
@@ -51,11 +51,11 @@ void selectItem(MenuItem item) {
             Intent intent = new Intent(context, CreateEventActivity.class);
             intent.putExtra(EVENT_ID, fragmentNavigator.getEventId());
             context.startActivity(intent);
-        } else if (id == R.id.nav_about_event) {
+        } /*else if (id == R.id.nav_about_event) {
             Intent intent = new Intent(context, AboutEventActivity.class);
             intent.putExtra(AboutEventActivity.EVENT_ID, fragmentNavigator.getEventId());
             context.startActivity(intent);
-        } else if (id == R.id.nav_share) {
+        }*/ else if (id == R.id.nav_share) {
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/main/DrawerNavigator.java]: Remove the code

File: app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java [hunk 2]
```diff
@@ -47,7 +47,7 @@
         R.id.nav_dashboard,
         R.id.nav_attendees,
         R.id.nav_share,
-        R.id.nav_about_event,
+        //R.id.nav_about_event,
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java]: Remove the code

File: app/src/main/res/menu/activity_main_drawer.xml [hunk 3]
```diff
@@ -96,11 +96,11 @@
             android:title="@string/speakers_call"
             android:visible="false" />
 
-        <item
+        <!--<item
             android:id="@+id/nav_about_event"
             android:icon="@drawable/ic_info"
             android:title="@string/about"
-            android:checkable="false" />
+            android:checkable="false" />-->
```
Comment (iamareebjamal) [inline app/src/main/res/menu/activity_main_drawer.xml]: Same

File: app/src/main/java/com/eventyay/organizer/core/main/DrawerNavigator.java (hunk 1 above)
Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/main/DrawerNavigator.java]: Updated.

File: app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java (hunk 2 above)
Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java]: Updated.

File: app/src/main/res/menu/activity_main_drawer.xml (hunk 3 above)
Comment (ShridharGoel) [reply app/src/main/res/menu/activity_main_drawer.xml]: Updated.

**Pull request merged**